src/types/feeBreakdown.ts

```diff
@@ -21,10 +23,14 @@ export type ReductionOrExemption =
 export interface PassportFeeFields {
   "application.fee.calculated": number;
   "application.fee.payable": number;
-  "application.fee.payable.includesVAT": boolean;
   "application.fee.reduction.alternative": boolean;
   "application.fee.reduction.parishCouncil": boolean;
   "application.fee.reduction.sports": boolean;
   "application.fee.exemption.disability": boolean;
   "application.fee.exemption.resubmission": boolean;
+  "application.fee.serviceCharge"?: number;
+  "application.fee.serviceCharge.VAT"?: number;
+  "application.fee.fastTrack"?: number;
+  "application.fee.fastTrack.VAT"?: number;
+  // `${string}.VAT`?: string;
```

Comment:
Haven't gotten this dynamic type working yet - see this skipped test with example key ! https://github.com/theopensystemslab/planx-core/pull/694/files#diff-935434782beb09636d1698cb7deddb0d16142667ffb111db3736ca2284aeb77cR114